pkgs/development/interpreters/octave/build-octave-package.nix

```diff
@@ -62,13 +62,18 @@ let
   ]
   ++ nativeBuildInputs;
 
+  passthru' = {
+    updateScript = ../../../../maintainers/scripts/update-octave-packages;
```

Comment:
Does the script automatically gets the `default.nix` of the expression's path?

Reply:
I think so? `update-octave-packages` is a `#!/usr/bin/env nix-shell` script that calls out to the `update-octave-shell.nix` environment. I am not sure the best way to get the script to be picked up.

Reply:
> > Does the script automatically gets the `default.nix` of the expression's path?
> I think so? `update-octave-packages` is a `#!/usr/bin/env nix-shell` script that calls out to the `update-octave-shell.nix` environment. I am not sure the best way to get the script to be picked up.

I'm afraid there is a problem with this passthru... `updateScript` doesn't get the package's `default.nix` as argument with this:

```sh
nix-shell maintainers/scripts/update.nix --argstr package octavePackages.statistics --argstr commit true
```

Fails.

Reply:
I'm working on a solution using `os.environ.UPDATE_NIX_ATTR_PATH`.